app/models/user_mode/auth0_auth_strategy.rb

```diff
@@ -0,0 +1,22 @@
+class UserMode::Auth0AuthStrategy < UserMode::MultiUser
+  def set_auth_provider(omniauth)
+    omniauth.provider :auth0,
+                      Rails.configuration.auth0_client_id,
+                      Rails.configuration.auth0_client_secret,
```

Comment:
Although the last option looks simpler, I still would like to keep the abstraction of `single_user`, since there may be another customizations that have only sense when you have only one user - not only auth settings. 

That is why I still prefer the SingleUser/MultiUser concept